### PR TITLE
Dev/pcb

### PR DESCRIPTION
**Pull request by cychitivav**
Kicad files for PCB

### CODE REVIEW

File: docs/dual-max14870-motor-driver-shield-for-arduino-dimensions.pdf
Comment (jsduenass) [inline docs/dual-max14870-motor-driver-shield-for-arduino-dimensions.pdf:1]: This driver is no longer used. And using pdf int the documentation does not seem as agood practice.

File: kicad/#auto_saved_files#
Comment (jsduenass) [inline kicad/#auto_saved_files#:1]: we should gitginore this file

File: kinematics.py
Comment (jsduenass) [inline kinematics.py:1]: File not relevant to this pull request

File: kinematics_damping_system.m
Comment (jsduenass) [inline kinematics_damping_system.m:1]: File not relevant to pull request

**Review by jsduenass — CHANGES_REQUESTED** [review]

**Comment by cychitivav** [comment]
I'm still not sure whether to keep the gerbers in the git history.

**Review by jsduenass — APPROVED** [review]
Looks good

**Pull request merged**